### PR TITLE
zile: add page

### PR DESCRIPTION
**Pull request by anoopemacs**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/zile.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# zile
+
+> Zile is a lightweight clone of the Emacs text editor.
+> More information: <https://www.gnu.org/software/zile/>.
+
+- Start in console mode (without X window):
+
+`zile -nw`
```
Comment (navarroaxel) [inline pages/linux/zile.md:8]: Why I'm getting this message `Unknown option '-nw'`?

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hi @anoopemacs, welcome and thanks for your submission. I left a comment.

And please **don't** push force commits to the PR, is harder for us to track the changes and comments.

File: pages/linux/zile.md (hunk 1 above)
Comment (anoopemacs) [reply pages/linux/zile.md]: Zile is a console only editor. Hence, -w was redundant. Have removed it now.

Comment (navarroaxel) [reply pages/linux/zile.md]: I tried `-n` too but: `Unknown option '-n'`

Comment (anoopemacs) [reply pages/linux/zile.md]: Have removed them both now. Apologies for the silly error.

File: pages/linux/zile.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# zile
+
+> Zile is a lightweight clone of the Emacs text editor.
+> More information: <https://www.gnu.org/software/zile/>.
+
+- Start in console mode (without X window):
```
Comment (navarroaxel) [inline pages/linux/zile.md:6]: If console-only app, maybe this is better:
```suggestion
- Start a buffer for volatile notes, they won't be saved:
```

File: pages/linux/zile.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# zile
+
+> Zile is a lightweight clone of the Emacs text editor.
+> More information: <https://www.gnu.org/software/zile/>.
+
+- Start a buffer for volatile notes, they won't be saved:
```
Comment (navarroaxel) [inline pages/linux/zile.md:6]: Oh, I prefer your word:
```suggestion
- Start a buffer for temporary notes, they won't be saved:
```

Comment (anoopemacs) [reply pages/linux/zile.md]: Done :)

**Review by navarroaxel — APPROVED** [review]

File: pages/linux/zile.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+# zile
+
+> Zile is a lightweight clone of the Emacs text editor.
+> More information: <https://www.gnu.org/software/zile/>.
+
+- Start a buffer for temporary notes, they won't be saved:
```
Comment (sbrl) [inline pages/linux/zile.md:6]: Perhaps this will read better?
```suggestion
- Start a buffer for temporary notes, which won't be saved:
```

**Review by sbrl — COMMENTED** [review]
Thanks for the new page! I've left a comment below for you to review.

File: pages/linux/zile.md (hunk 4 above)
Comment (anoopemacs) [reply pages/linux/zile.md]: Yes, that sounds better grammatically.

**Review by sbrl — APPROVED** [review]
Thanks, @anoopemacs!

**Pull request merged**